packages/auto-utils/src/number.ts

```diff
@@ -296,7 +296,7 @@ export const shannonsToAi3 = (
  * This function accepts AI3 amounts as strings and performs string-to-BigInt conversion.
  * For direct Shannon (BigInt) validation, use `meetsExistentialDepositShannons` instead.
  *
- * For Autonomys Network, the existential deposit is 0.000001 AI3 (1,000,000,000,000 Shannons).
+ * For Autonomys Network, the existential deposit is 0.00001 AI3 (10,000,000,000,000 Shannons).
```

Comment:
I understand these will be split into multiple methods, but the comments should have said consensus (or domains) to start with:
```suggestion
 * For Autonomys Network consensus chain, the existential deposit is 0.00001 AI3 (10,000,000,000,000 Shannons).
```